  
<launch>
    <!-- Start Action Server that moves the robot -->
    <node pkg="spelling_action_server" type="spelling_action_server_src.py" name="spelling_action_server_node"  output="screen">
    </node>
</launch>